xml/System.IO.Compression/BrotliEncoder.xml

```diff
@@ -142,10 +151,10 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, a span of bytes where the compressed data will be stored.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes that were written to <paramref name="destination" />.</param>
+        <summary>Compresses an empty read-only span of bytes into <paramref name="destination" />, ensuring output is produced for all the processed input. An actual flush is performed when the source is depleted and there is enough space in <paramref name="destination" />.</summary>
```

Comment:
The [source code](https://github.com/dotnet/corefx/blob/c668e96cb0886d13a051d8021a3c3147e7daa15c/src/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliEncoder.cs#L113) for `Flush` internally calls `Compress` with an empty source parameter. I wrote a very literal description of what this method does, but I don't really understand the true purpose. Can you please help me improve this description, @ahsonkhan?